content/docs/how-to-guides/using-branches.md

```diff
@@ -0,0 +1,52 @@
+---
+title: How to Use Branches
+---
+
+Now that you have access to [branching](https://neon.tech/docs/conceptual-guides/branching.md) on Neon, let's look at how you can use it.
```

Comment:
let's add a placeholder for those who discover this page without a having a branching permission
users can get early access to branching by sending an email and describing their use case